app/models/request.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class Request < ApplicationRecord
-  validates_presence_of :title, :description, :reward
+  validates_presence_of :title, :description, :reward, :status
   belongs_to :requester, class_name: 'User'
   has_many :offers
+  enum status: { pending: 0, started: 1, active: 2, completed: 3 }
```

Comment:
Isn't it pending -> active -> started -> completed?
But it doesn't matter too much, as long as we can keep the unique values

Reply:
done done